source/crud/write-operations/arrayFilters.txt

```diff
@@ -0,0 +1,112 @@
+===========================
+Update Arrays in a Document
+===========================
+
+.. default-domain:: mongodb
+
+To modify arrays nested within a document, use the
+:manual:`filtered positional operator
+</reference/operator/update/positional-filtered/index.html>`. You can
+use this operator in any :doc:`update document
+</crud/write-operations/update-document` to modify positions in an array
+matched by conditions passed to the query via the ``arrayFilters``
+option.
+
+You can think of ``arrayFilters`` as an additional query level applied
+only to documents already matched by the update operation's query filter
+(the first parameter for ``updateOne()`` and ``updateMany()``). This
+query finds all positions in the referenced array that match your
+criteria, and you can use the returned position values to modify only
+those positions.
```

Comment:
- Rather than "positions in an array" should we say "specific elements in an array"? "Position values" and such aren't super clear terms and make this a little more confusing for me.
- The conditions are query expressions. I think it's important to convey that you're essentially running a subquery on the array elements. Thoughts?
- The "additional query level" framing doesn't clarify much IMO.

Reply:
- reworded
- added mention of subquery
- personally i think the first paragraph, while technically correct, is complete gibberish to anyone who doesn't already know what a query filter is. Or is at least overwhelming. The second paragraph is a lot more comprehensible to me (and when I inevitably forget what an arrayFilter is in 3 months I'll come back to this page and have a much better understanding after that paragraph)